admin_guide/router.adoc

```diff
@@ -0,0 +1,54 @@
+= Routers
```

Comment:
Would it make sense to title this "Monitoring Routers" / `monitoring_routers.adoc`?